### PR TITLE
IntelliSense fixes - PR 14/?

### PR DESCRIPTION
**Pull request by mairaw**
- Adds missing descriptions
- Adds dropped exception conditions during migration
- Style guide fixes
- Minor fixes

Contributes to #1330 and #1358

### CODE REVIEW

File: xml/System.Net.WebSockets/ClientWebSocket.xml [hunk 1]
```diff
@@ -529,9 +515,7 @@
       </ReturnValue>
       <Docs>
         <summary>Get the WebSocket state of the <see cref="T:System.Net.WebSockets.ClientWebSocket" /> instance.</summary>
```
Comment (rpetrusha) [inline xml/System.Net.WebSockets/ClientWebSocket.xml]:
```suggestion
        <summary>Gets the WebSocket state of the <see cref="T:System.Net.WebSockets.ClientWebSocket" /> instance.</summary>
```

File: xml/System.Net.WebSockets/WebSocket.xml [hunk 2]
```diff
@@ -636,9 +626,7 @@
         <param name="buffer">References the application buffer that is the storage location for the received data.</param>
         <param name="cancellationToken">Propagates the notification that operations should be canceled.</param>
         <summary>Receives data from the <see cref="T:System.Net.WebSockets.WebSocket" /> connection asynchronously.</summary>
-        <returns>Returns <see cref="T:System.Threading.Tasks.Task`1" />.  
-  
- The task object representing the asynchronous operation. The <see cref="P:System.Threading.Tasks.Task`1.Result" /> property on the task object returns a <see cref="T:System.Byte" /> array containing the received data.</returns>
+        <returns>The task object representing the asynchronous operation. The <see cref="P:System.Threading.Tasks.Task`1.Result" /> property on the task object returns a <see cref="T:System.Byte" /> array containing the received data.</returns>
```
Comment (rpetrusha) [inline xml/System.Net.WebSockets/WebSocket.xml]: Possibly the detail about the Result property is better suited for remarks.

File: xml/System.Web/AspNetHostingPermission.xml [hunk 3]
```diff
@@ -171,7 +171,7 @@
           <paramref name="securityElement" /> is <see langword="null" />.</exception>
         <exception cref="T:System.ArgumentException">The <see cref="P:System.Security.SecurityElement.Tag" /> property of <paramref name="securityElement" /> is not equal to "IPermission".  
   
- <see langword="-" /> or-  
+ -or-
```
Comment (rpetrusha) [inline xml/System.Web/AspNetHostingPermission.xml]:
```suggestion
-or-  
```

File: xml/System.Windows.Forms/ListBox+ObjectCollection.xml [hunk 4]
```diff
@@ -415,7 +415,7 @@
       </Parameters>
       <Docs>
         <param name="destination">The object array in which the items from the collection are copied to.</param>
-        <param name="dest">To be added.</param>
+        <param name="dest">The object array in which the items from the collection are copied to.</param>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/ListBox+ObjectCollection.xml]:
```suggestion
        <param name="dest">The object array to which the items from the collection are copied.</param>
```

File: xml/System.Windows.Forms/ListBox+SelectedObjectCollection.xml [hunk 5]
```diff
@@ -208,7 +208,7 @@
       </Parameters>
       <Docs>
         <param name="destination">An <see cref="T:System.Array" /> representing the array to copy the contents of the collection to.</param>
-        <param name="dest">To be added.</param>
+        <param name="dest">An <see cref="T:System.Array" /> representing the array to copy the contents of the collection to.</param>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/ListBox+SelectedObjectCollection.xml]:
```suggestion
        <param name="dest">An array to copy the contents of the collection to.</param>
```

File: xml/System.Windows.Forms/ListBox+SelectedObjectCollection.xml [hunk 6]
```diff
@@ -208,7 +208,7 @@
       </Parameters>
       <Docs>
         <param name="destination">An <see cref="T:System.Array" /> representing the array to copy the contents of the collection to.</param>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/ListBox+SelectedObjectCollection.xml]:
```suggestion
        <param name="destination">An array to copy the contents of the collection to.</param>
```

File: xml/System.Windows.Forms/ListBox+ObjectCollection.xml [hunk 7]
```diff
@@ -415,7 +415,7 @@
       </Parameters>
       <Docs>
         <param name="destination">The object array in which the items from the collection are copied to.</param>
```
Comment (mairaw) [inline xml/System.Windows.Forms/ListBox+ObjectCollection.xml]:
```suggestion
        <param name="destination">The object array to which the items from the collection are copied.</param>
```

File: xml/System.Net.WebSockets/WebSocket.xml (hunk 2 above)
Comment (mairaw) [reply xml/System.Net.WebSockets/WebSocket.xml]: I'm leaving these alone for now.

**Pull request merged**